docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -136,6 +145,34 @@ By default, the Jira Teleport plugin will use a config in `/etc/teleport-jira.to
 </TabItem>
 </Tabs>
 
+<Details title="Running Teleport in Kubernetes?">
+  <Tabs>
+    <TabItem label="Self-hosted" scope={["oss","enterprise","cloud"]}>
+
```

Comment:
We need a sentence or two to put the YAML content below into context, i.e., say it's a values file and we're using it to configure the Helm chart. (This applies to both tabs.)

Reply:
I've added a short explanation and a link for the reference docs. What do you think?